state/logs.go

```diff
@@ -15,13 +15,13 @@ import (
 
 	"github.com/dustin/go-humanize"
 	"github.com/juju/clock"
+	"github.com/juju/collections/deque"
```

Comment:
Is this correct?

Reply:
This package probably got moved to `juju/collections` together with `set`. It is not currently present in `juju/utils`.